src/routes/settings/(nav)/assistants/[assistantId]/edit/+page.server.ts

```diff
@@ -20,6 +21,9 @@ const newAsssistantSchema = z.object({
 	exampleInput3: z.string().optional(),
 	exampleInput4: z.string().optional(),
 	avatar: z.union([z.instanceof(File), z.literal("null")]).optional(),
+	ragLinkList: z.preprocess(parseStringToList, z.string().url().array().max(10)),
```

Comment:
is there anywhere we are checking that links are valid URL somewhere ?

Reply:
zod already does some validation, and then [we resolve the domain](https://github.com/huggingface/chat-ui/blob/940edd02a66c4c9b5e4c8edb92a78a7b15993c3e/src/lib/server/isURLLocal.ts#L1-L27) but we don't check the url itself to see if it's valid